web/scm/src/app/big-query-service/big-query-service.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { environment } from '../../environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BigQueryService {
```

Comment:
Cool, but I think we will have a lot of services, can we name parent directory services instead of big-query-service and we can put all other services there, what do you think?